blog/views.py

```diff
@@ -34,4 +34,10 @@ def get(self, request, detail_post_slug, *args, **kwargs):
 class PostViewSet(viewsets.ModelViewSet):
     serializer_class = PostSerializer
     queryset = Post.objects.all()
-    http_method_names = ['get', 'head', 'list']
\ No newline at end of file
+    http_method_names = ['get', 'head', 'list']
+
+
+class PostDetailViewSet(viewsets.ModelViewSet):
+    serializer_class = PostDetailsSerializers
+    queryset = Post.objects.all()
```

Comment:
Post detail should be querying for a single item via url parameters:

```
# Show one account
    path("accounts/<str:uuid>",UserProfileDetailView.as_view(),name="single-account"),
```

Mixing (PostView, ListView etc) can be implemented instead of making another Class

```
class UsersLocationView(UpdateAPIView, ListAPIView):
...
```